auth-web/src/components/auth/Signin.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+
+</template>
+
+<script lang="ts">
+import { Vue, Component, Prop } from 'vue-property-decorator'
+import { getModule } from 'vuex-module-decorators'
+import commonUtils from '../../util/common-util'
+import UserModule from '../../store/modules/user'
+
+@Component
+export default class Signin extends Vue {
+  private userStore = getModule(UserModule, this.$store)
+
+  @Prop({ default: 'bcsc' })
+  idpHint: string
+
+  @Prop()
+  redirectUrl: string
+
+  mounted () {
+    this.userStore.initKeycloak(this.idpHint).then((kcInit) => {
+      kcInit.success((authenticated) => {
+        if (authenticated === true) {
+          this.userStore.initializeSession().then(() => {
+            // If a redirect url is given, redirect to that page else continue to dashboard or userprofile
+            if (this.redirectUrl) {
+              if (commonUtils.isUrl(this.redirectUrl)) {
+                setTimeout(() => {
```

Comment:
is there a need of setTimeOut? We can just redirect?

Reply:
Done